strawberry_django/subscriptions/receivers.py

```diff
@@ -0,0 +1,12 @@
+import channels.layers
+from asgiref.sync import async_to_sync
+
+from .utils import get_group, get_msg
+
+
+def refresh_subscription_receiver(instance):
+    group = get_group(instance)
+    msg = get_msg(instance)
+
+    channel_layer = channels.layers.get_channel_layer()
+    async_to_sync(channel_layer.group_send)(group=group, message=msg)
```

Comment:
Maybe this should be `async` and the caller itself should use `async_to_sync`?

Reply:
I'll need to play with it. On that makes sense, since we're looking at async first.